src/pps/pps.proto

```diff
@@ -711,21 +711,28 @@ message CreatePipelineRequest {
   string salt = 22;
   int64 datum_tries = 23;
   SchedulingSpec scheduling_spec = 24;
-  string pod_spec = 25; // deprecated, use pod_patch below
-  string pod_patch = 26; // a json patch will be applied to the pipeline's pod_spec before it's created;
+  string pod_spec = 25;; // deprecated, use pod_patch below
+  string pod_patch = 26;; // a json patch will be applied to the pipeline's pod_spec before its created;
   pfs_v2.Commit spec_commit = 27;
   Metadata metadata = 28;
   string reprocess_spec = 29;
   bool autoscaling = 30;
   repeated Toleration tolerations = 34;
   ResourceSpec sidecar_resource_requests = 35;
-  string details_json = 36;
-  bool dry_run = 37;
   Determined determined = 38;
+  reserved 36, 37;
```

Comment:
No need to reserve fields that only ever existed in unstable (alpha) releases.  I guess the plan though is for those to go out with 2.7 and this to be merged after 2.7 is cut?

Reply:
> I guess the plan though is for those to go out with 2.7 and this to be merged after 2.7 is cut?

Yup. Kinda annoying, but I think it’s a good idea to keep good proto discipline.